crates/worker/src/partition_processor_manager.rs

```diff
@@ -841,37 +1039,29 @@ impl<T: TransportConnect> PartitionProcessorManager<T> {
             invoker.handle(),
         );
 
-        // the name is also used as thread names for the corresponding tokio runtimes, let's keep
-        // it short.
-        let task_name = self
-            .name_cache
-            .entry(partition_id)
-            .or_insert_with(|| Box::leak(Box::new(format!("pp-{}", partition_id))));
-
-        let invoker_name = Box::leak(Box::new(format!("invoker-{}", partition_id)));
-        let invoker_config = self.updateable_config.clone().map(|c| &c.worker.invoker);
-        let configuration = self.updateable_config.clone();
+        let invoker_name = Box::leak(Box::new(format!("invoker-{}", self.partition_id)));
+        let invoker_config = self.configuration.clone().map(|c| &c.worker.invoker);
 
-        let task_center = self.task_center.clone();
-        let maybe_task_id: Result<TaskId, RuntimeError> = self.task_center.start_runtime(
+        let tc = task_center();
+        let maybe_task_id: Result<TaskId, RuntimeError> = task_center().start_runtime(
```

Comment:
can we use `tc` instead of `task_center()`? This avoids a task local variable read.